internal/pkg/apiutil/attribute.go

```diff
@@ -147,6 +151,116 @@ func NewClusterListAttributeFromNative(a *bgp.PathAttributeClusterList) *api.Clu
 	}
 }
 
+func NewPrefixSIDAttributeFromNative(a *bgp.PathAttributePrefixSID) *api.PrefixSID {
+	psid := &api.PrefixSID{}
+	psid.Tlvs = MarshalSRv6TLVs(a.TLVs)
+
+	return psid
+}
+
+func MarshalSRv6TLVs(tlvs []bgp.PrefixSIDTLVInterface) []*any.Any {
+	mtlvs := make([]*any.Any, 0)
```

Comment:
it is possible to preallocate mtlvs here, like:
```
mtlvs := make([]*any.Any, 0, len(tlvs))
```
so that no reallocations are made inside the loop.

Reply:
done